test/AutobahnTestApp/AutobahnTestApp.csproj

```diff
@@ -1,16 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  
```

Comment:
Don't mess with the style. The extra blank lines just makes your actual changes harder to see.